src/Stratis.Bitcoin.Features.Wallet/Broadcasting/FullNodeBroadcasterManager.cs

```diff
@@ -22,24 +23,22 @@ public FullNodeBroadcasterManager(IConnectionManager connectionManager, IMempool
         }
 
         /// <inheritdoc />
-        public override async Task<bool> TryBroadcastAsync(Transaction transaction)
+        public override void BroadcastTransaction(Transaction transaction)
         {
             if (transaction == null)
                 throw new ArgumentNullException(nameof(transaction));
 
             if (this.IsPropagated(transaction))
-                return true;
+                return;
 
             var state = new MempoolValidationState(false);
-            if (!await this.mempoolValidator.AcceptToMemoryPool(state, transaction).ConfigureAwait(false))
+            if (!this.mempoolValidator.AcceptToMemoryPool(state, transaction).GetAwaiter().GetResult())
```

Comment:
why is this not async anymore?

Reply:
Because now BroadcastTransaction method don't have a loop that would wait for tx to be propagated back so I've changed BroadcastTransactionAsync to BroadcastTransaction. And the only async method inside the BroadcastTransaction that was left is AcceptToMemoryPool so I looked at the only usage of BroadcastTransaction and we'd await it anyway so I changed it to be awaited inside of BroadcastTransaction by running it synchronously.